README.md

```diff
@@ -259,3 +215,19 @@ Publish:
 export AH_TOKEN=xxxxx 
 make publish
 ```
+
+## Support
+
+If you encounter issues or have questions, you can submit a support request through the following channels:
+ - Open a Support ticket with Red Hat
+ - GitHub Issues: Report bugs, request features, or ask questions by opening an issue in the [GitHub repository](https://github.com/openshift/network.offline_migration_sdn_to_ovnk).
```

Comment:
Sorry I didn't get to the review request on time. Validated content has no RH support so can you remove line 222 above pls?

Reply:
@samccann https://github.com/openshift/network.offline_migration_sdn_to_ovnk/pull/24